### DISCUSSION TITLE
4 kilobytes of Environment Variables is not enough

### DISCUSSION THREAD

**Discussion by ctrlaltdylan** [vercel/community#421] [Help] [answered]
I don't think it's uncommon to make use of Environment Variables to make apps more configurable and flexible.

4KBs of data is very easy to run through.

If a lambda is limited to 512 mb, why does Vercel enforce a 4 kb limit? 

4 kilobytes = 0.004 megabytes

Therefore the total space reserved for user defined environment variables is limited to 0.0007% of available space per lambda.

Can we have this increased? It seems a bit low...

**Comment by Pieparker** [comment]
Hey there! The 4kb limit cannot be increased at this time. [Check this out](https://aws.amazon.com/premiumsupport/knowledge-center/lambda-environment-variable-size/) for more information!

**Comment by MFCo** [comment, marked as answer]
Hi there! Wanted to update you on this issue. The limit has now been increased to 64KB. [Here is the changelog](https://vercel.com/changelog/16x-larger-environment-variable-storage-up-to-64kb)

**Reply by ctrlaltdylan** [reply]
Thanks @MFCo this is great news!

But the link is 404'ing for me, even when logged in.

**Reply by MFCo** [reply]
Sorry! Just fixed it 🙏
